packaging/amazon-linux-ami/ecs-init.spec

```diff
@@ -24,6 +24,7 @@ BuildRoot:      %{_tmppath}/%{name}-%{version}-%{release}-root-%(%{__id_u} -n)
 
 Source0:        sources.tgz
 Source1:        ecs.conf
+Source2:        https://s3.amazonaws.com/amazon-ecs-agent/ecs-agent-v%{bundled_agent_version}.tar
```

Comment:
Which entity is responsible for generating this object? Is that being done in of ECS agent's release/publish scripts?

Reply:
This is an RPM Source declaration, today we don't have a wrapper to fetch this artifact automatically. Standard tools like `spectool` can be used to read this and fetch the appropriate source.